scss/_popover.scss

```diff
@@ -11,8 +12,10 @@
   --#{$prefix}popover-inner-border-radius: #{$popover-inner-border-radius};
   --#{$prefix}popover-box-shadow: #{$popover-box-shadow};
   --#{$prefix}popover-header-padding-x: #{$popover-header-padding-x};
```

Comment:
We use it in:

```
$popover-body-padding-x:            $popover-header-padding-x !default;
```

Reply:
But the CSS var isn't used unless I'm mistaken ?
And I'm wondering if should let this kind of dependency between vars, I let you decide on this!

Reply:
You are right, I just deprecated the CSS var. But I think we can keep the SCSS var just to keep the number of differences with Boostrasp as low as possible 🤔

Reply:
OK with the deprecation of the CSS var as highlighted by LM. Its value must not change so remains `#{$popover-header-padding-x}` and we can't deprecate the Sass var since it is used by `$popover-body-padding-x`. So seems fine except if I missed something here.